x-pack/legacy/plugins/apm/public/components/app/ServiceMap/cytoscapeOptions.ts

```diff
@@ -26,6 +26,12 @@ const layout = {
   padding: nodeHeight
 };
 
+function getEdgeColor(el: cytoscape.EdgeSingular) {
+  return el.selected() || el.hasClass('hover') || el.hasClass('nodeHover')
```

Comment:
what is the difference between doing it this way and creating a new style for `.hover, .nodeHover`?

Reply:
Nothing really, but I think it's easier to think about it when it's all in one function vs. having multiple style objects and figuring out how they're ordered and override each other.

Reply:
that's fair